bccsp/handlers/nymsigner.go

```diff
@@ -35,6 +32,32 @@ func (s *NymSigner) Sign(k bccsp.Key, digest []byte, opts bccsp.SignerOpts) ([]b
 		return nil, errors.New("invalid issuer public key, expected *issuerPublicKey")
 	}
 
+	// handle the smartcard case
+	if signerOpts.IsSmartcard {
```

Comment:
this part of the code is not tested, can it be without an actual smartcard?

Reply:
yeah exactly, that's tricky. This particular code is only a wrapper to call 

https://github.com/IBM/idemix/blob/smartcard-squash/bccsp/schemes/aries/smartcard.go#L116

It's difficult to test that from the bccsp layer since it would require the creation of a mock bccsp smartcard signer. Otoh we _do_ test it from here using the emulated signing.

Reply:
actually I'm sorry - I wrote this code a while back so I forgot: I double-checked and we DO test this - just not from this package: In particular, this line

https://github.com/IBM/idemix/blob/smartcard-squash/bccsp/smartcard_test.go#L264

ends up testing exactly this code.

Reply:
perfect 👍